android/app/src/e2e/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config xmlns:tools="http://schemas.android.com/tools">
+	<base-config cleartextTrafficPermitted="false">
+		<trust-anchors>
+			<certificates src="system" />
+			<certificates src="user"
+				tools:ignore="AcceptsUserCertificates" />
+		</trust-anchors>
+	</base-config>
+	<domain-config cleartextTrafficPermitted="true">
+		<domain includeSubdomains="true">10.0.2.2</domain>
+		<domain includeSubdomains="true">localhost</domain>
```

Comment:
You might not need these lines, since `cleartextTrafficPermitted="true"`

Reply:
Aim was to just enable clearText on local hosts for detox. Can be disabled for all other servers still, so we just override the hostnames needed.